cmd/zetaclientd/start.go

```diff
@@ -240,6 +241,8 @@ func Start(_ *cobra.Command, _ []string) error {
 					ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 					defer cancel()
 					result := <-ping.Ping(ctx, host, p)
+					pingRTTLock.Lock()
```

Comment:
Why not doing it before `result := <-ping.Ping(ctx, host, p)`

LGTM in any case

Reply:
Because that blocks until the ping returns? We wouldn't want to be holding a lock if we aren't going to be doing anything?